File: Modul4/C/PRAK405-2310817310012-MUHAMMAD_AZWIN_HAKIM.c
#include <stdio.h>

int main() {
    int n, k;

    scanf("%d %d", &n, &k);

    int m[n][n];

    for (int i = 0; i < n; i++) {
        for (int j = 0; j <= i; j++) {
            m[i][j] = (i - j + 1) * k;
            printf("(%d * %d) ", i - j + 1, k);
        }

        printf("= ");

        int jpb = 0;
        for (int j = 0; j <= i; j++) {
            jpb += m[i][j];
        }

        printf("%d\n", jpb);
    }

    int total = 0;
    for (int i = 0; i < n; i++) {
        for (int j = 0; j <= i; j++) {
            total += m[i][j];
        }
    }
    printf("%d\n", total);

    return 0;
}

File: Modul4/Python/PRAK405-2310817310012-MUHAMMAD_AZWIN_HAKIM.py
n, k = map(int, input().split())

m = [[0] * n for _ in range(n)]

for i in range(n):
    for j in range(i + 1):
        m[i][j] = (i - j + 1) * k
        print(f"({i - j + 1} * {k}) ", end='')

    print("= ", end='')

    jpb = sum(m[i][:i + 1])
    print(jpb)

total = sum(sum(row[:i + 1]) for i, row in enumerate(m))
print(total)
